lnwire/channel_ready.go

```diff
@@ -5,15 +5,24 @@ import (
 	"io"
 
 	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/lightningnetwork/lnd/fn"
 	"github.com/lightningnetwork/lnd/tlv"
 )
 
 const (
+	// ChanReadyAnnounceBtcNonce is the TLV type associated with the
+	// bitcoin nonce field in the channel_ready message.
+	ChanReadyAnnounceBtcNonce = tlv.Type(0)
+
 	// ChanReadyAliasScidType is the TLV type of the experimental record of
 	// channel_ready to denote the alias being used in an option_scid_alias
 	// channel.
 	ChanReadyAliasScidType = tlv.Type(1)
 
+	// ChanReadyAnnounceNodeNonce is the TLV type associated with the node
+	// nonce field in the channel_ready message.
+	ChanReadyAnnounceNodeNonce = tlv.Type(2)
```

Comment:
in the spec this one has a type value of 0 and the `ChanReadyAnnounceBtcNonce` has a val of 2

Reply:
fixed here :)